themes/mlibrary_new/exhibit-builder/exhibits/show.php

```diff
@@ -61,37 +61,43 @@
       </h2>
 
       <div>
-        <?php echo exhibit_builder_render_exhibit_page($exhibit_page); ?>
+        <?php exhibit_builder_render_exhibit_page($exhibit_page);?>
       </div>
-  </div>
-  
+      <?php $children = $exhibit_page->getChildPages();
+               if ($children) {
+                 foreach ($children as $child) {?>
+                   <h3> <?php echo metadata($child, 'title');?></h3>
+                    <div> <?php echo exhibit_builder_render_exhibit_page($child);?> </div>
+                    <?php release_object($child);
+                 }
+              }
+         ?>
+  </div> 
   <div class="col-xs-12 col-sm-9 col-sm-offset-3 padding-0">
     <div class="section-nav">
-          <div class="section-nav-panel">
-            <div class="section-nav-image"></div>
-            <div class="section-link-prev">
-              <?php
-                echo exhibit_builder_link_to_previous_page("Previous Section");
-              ?>
-            </div>
-            <div class="section-nav-text">
-              <p class="section-title">Section Title</p>
-              <p class="section-caption"></p>
-            </div>
-          </div>
-
-          <div class="section-nav-panel">
-            <div class="section-nav-image"></div>
-              <div class="section-link-next">
-              <?php
-                echo exhibit_builder_link_to_next_page("Next Section");
-              ?>
-            </div>
-            <div class="section-nav-text">
-              <p class="section-title">Section Title</p>
-              <p class="section-caption"></p>
-            </div>
-          </div>
+          <?php $navigate_previous_exhibit = mlibrary_new_exhibit_builder_previous_link_to_exhibit($exhibit, $exhibit_page);?>
+          <?php if (!empty($navigate_previous_exhibit)){ ?>  
+                    <div class="section-nav-panel">
```

Comment:
Is there a way you could restructure 80-87 to reflect semantic relationships among these elements?

Reply:
Good point, I will create a separate issue to look at it while finalizing the page style.

Reply:
I tried to add some semantics (nav, figure, a section from the falafel.css but the current style structured with div so it did not display the information well). I will create a new issue to add semantic to the next and previous navigation.